components/NotesCard.vue

```diff
@@ -144,6 +141,25 @@
         <template #tooltip>Bookmark</template>
       </VxTooltip>
     </div>
+  </div>
+  </div>
+  <div class="image-container" :style="preview ? '' : 'display:none;'">
+            <img
+          :src="note.images[0]"
+          v-if="hasImage && preview"  
+          style="position:absolute;object-fit: cover;min-height:100%"
+        />
+        <img
+          src="../assets/images/Quento_Physics_Filler.png"
+          v-if="!hasImage && preview"  
+          style="position:absolute;object-fit: cover;min-height:100%"
+        />
+
```

Comment:
Can we remove the preview image on mobile? 
![image](https://user-images.githubusercontent.com/46302202/92047039-1560fb80-ed52-11ea-96bc-23ec80290aff.png)
takes up too much space

Reply:
Also turn down the max-height on the md preview on mobile? 
![image](https://user-images.githubusercontent.com/46302202/92047084-2f9ad980-ed52-11ea-89b7-88c5564049da.png)
So either we show an image or md preview not both, saves alot of space

Reply:
Also theres this issue with the menu dots 
![image](https://user-images.githubusercontent.com/46302202/92047220-93250700-ed52-11ea-9788-199cce87c16c.png)
![image](https://user-images.githubusercontent.com/46302202/92047236-9b7d4200-ed52-11ea-9d24-50bb7372ea54.png)